.github/CONTRIBUTING.md doesn't work

<!--
Thank you for reporting an issue.

Please fill in as much of the template below as you're able.

Version:      if you know it, otherwise use the git revision
Installation: How did you install wallabag? Using git clone, the docker image, an installer, downloading the package, etc.
PHP version:  The version of PHP you are using
OS:           The host running wallabag
Database:     The storage system your instance is using (SQLite, MySQL/MariaDB or PostgreSQL) with the version
Parameters:   Paste the content of your app/config/parameters.yml (hide sensitive stuff if you want)
-->
### Environment

* **Version**: master (c684ad2d)
* **Installation**: Using .github/CONTRIBUTING.md Docker instructions
* **PHP version**: 
* **OS**: Ubuntu
* **Database**: sqlite
* **Parameters**:

<details>
  <summary>My <code>app/config/parameters.yml</code> is:</summary>

  ```
parameters:
    database_driver: pdo_sqlite
    database_driver_class: ~
    database_host: 127.0.0.1
    database_port: ~
    database_name: symfony
    database_user: root
    database_password: ~
    database_path: %kernel.project_dir%/data/db/wallabag.sqlite
    database_table_prefix: wallabag_
    database_socket: null
    database_charset: utf8

    domain_name: http://localhost:8000
    server_name: "Your wallabag instance"

    mailer_transport:  smtp
    mailer_user:       ~
    mailer_password:   ~
    mailer_host:       127.0.0.1
    mailer_port:       25
    mailer_encryption: ~
    mailer_auth_mode:  ~

    locale:            en

    # A secret key that's used to generate certain security-related tokens
    secret: ch4n63m31fy0uc4n

    # two factor stuff
    twofactor_auth: true
    twofactor_sender: no-reply@wallabag.org

    # fosuser stuff
    fosuser_registration: true
    fosuser_confirmation: true

    fos_oauth_server_access_token_lifetime: 3600
    fos_oauth_server_refresh_token_lifetime: 1209600

    from_email: wallabag@example.com

    rss_limit: 50

    # RabbitMQ processing
    rabbitmq_host: localhost
    rabbitmq_port: 5672
    rabbitmq_user: guest
    rabbitmq_password: guest
    rabbitmq_prefetch_count: 10

    # Redis processing
    redis_scheme: tcp
    redis_host: redis
    redis_port: 6379
    redis_path: ~
    redis_password: ~

    # Sentry
    sentry_dsn: ~

    session_handler: ~

  ```
</details>

### What steps will reproduce the bug?

<!--
Enter details about your bug and how to reproduce it
-->
Executing .github/CONTRIBUTING.md Docker instructions result in a exception, which causes the installation to be aborted:
```
Script Incenteev\ParameterHandler\ScriptHandler::buildParameters handling the post-cmd event terminated with an exception

                                                                                                    
  [Symfony\Component\Yaml\Exception\ParseException]                                                 
  The reserved indicator "%" cannot start a plain scalar; you need to quote the scalar at line 9 (  
  near "database_path: %kernel.project_dir%/data/db/wallabag.sqlite").                              
                                                                                                    
```

It seems that `database_path` in parameters.yml is not correctly escaped. I was not able to find how this line is created.

### Second issue
When the `database_path` is fixed, the composer install step will ask for:
```
Some parameters are missing. Please provide them.
mailer_dsn ('smtp://127.0.0.1'): 
```
I think this should have a default value.

#6174 fixes the `composer install` step. However the `docker-compose up` still has issues. The log ends with:
```
php_1    | 
php_1    |  [ERROR] The document root directory "/var/www/html/public" does not exist.     
php_1    | 
wallabag_php_1 exited with code 1
```

A workaround is `ln -sv web public`, but that doesn't seem like a permanent solution.

--- a/docker/php/Dockerfile
+++ b/docker/php/Dockerfile
@@ -53,7 +53,7 @@ RUN docker-php-ext-install -j "$(nproc)" \
 
 RUN pecl install redis; \
     pecl install imagick; \
-    pecl install xdebug; \
+    pecl install xdebug-3.1.6; \
     docker-php-ext-enable \
         redis \
         imagick \

--- a/docker/php/config/parameters.yml
+++ b/docker/php/config/parameters.yml
@@ -6,7 +6,7 @@ parameters:
     database_name: ${DATABASE_NAME:-symfony}
     database_user: ${DATABASE_USER:-root}
     database_password: ${DATABASE_PASSWORD:-~}
-    database_path: ${DATABASE_PATH:-"%kernel.root_dir%/data/db/wallabag.sqlite"}
+    database_path: '${DATABASE_PATH:-"%kernel.root_dir%/data/db/wallabag.sqlite"}'
     database_table_prefix: wallabag_
     database_socket: null
     database_charset: ${DATABASE_CHARSET:-utf8}
@@ -14,15 +14,9 @@ parameters:
     domain_name: ${DOMAIN_NAME:-https://www.example.com}
     server_name: ${SERVER_NAME:-"Your wallabag instance"}
 
-    mailer_transport:  ${MAILER_TRANSPORT:-smtp}
-    mailer_user:       ${MAILER_USER:-~}
-    mailer_password:   ${MAILER_PASSWORD:-~}
-    mailer_host:       ${MAILER_HOST:-127.0.0.1}
-    mailer_port:       ${MAILER_PORT:-25}
-    mailer_encryption: ${MAILER_ENCRYPTION:-~}
-    mailer_auth_mode:  ${MAILER_AUTH_MODE:-~}
+    mailer_dsn: ${MAILER_DSN:-"smtp://127.0.0.1"}
 
-    locale:            ${LOCALE:-en}
+    locale: ${LOCALE:-en}
 
     # A secret key that's used to generate certain security-related tokens
     secret: ${SECRET:-~}

